crates/driver/src/domain/competition/solution/encoding.rs

```diff
@@ -186,21 +185,8 @@ pub fn tx(
         // Necessary pre-interactions get prepended to the settlement. So to initiate
         // the loans in the desired order we need to add them in reverse order.
         .rev()
-        .map(|flashloan| {
-            // TODO add configuration options
-            // Hardcoded configuration for now
-            let maker_lender = addr!("60744434d6339a6B27d73d9Eda62b6F66a0a04FA");
-            let aave_lender = addr!("87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2");
-
-            let (flashloan_wrapper, flash_fee_bps) = if flashloan.lender.0 == maker_lender {
-                (&contracts.flashloan_wrappers()[0], eth::U256::zero()) // MAKER
-            } else if flashloan.lender.0 == aave_lender {
-                // TODO: ask AAVE to waive the current 5 BPS fee for our helper contract
-                (&contracts.flashloan_wrappers()[1], eth::U256::from(5)) // AAVE
-            } else {
-                // TODO remove this together with configuration options
-                (&contracts.flashloan_wrappers()[0], eth::U256::zero()) // for driver tests to pass
-            };
+        .filter_map(|flashloan| {
+            let flashloan_wrapper = contracts.flashloan_wrapper_by_lender().get(&flashloan.lender)?;
```

Comment:
I think we should probably return an error here. Otherwise it would be hard to figure out that solutions get dropped because a solver wants to use an unsupported lender.